common/main/java/com/couchbase/lite/Collection.java

```diff
@@ -7,21 +7,13 @@
 import java.util.List;
 import java.util.concurrent.Executor;
 
-import com.couchbase.lite.internal.utils.Preconditions;
-
 
 public class Collection implements Indexable, DatabaseChangeObservable {
-    private final String name;
-    private final int count;
-    private final Scope scope;
+    private static final String COLLECTION_NAME = "_default";
```

Comment:
I think you probably want to be public and named "DEFAULT_COLLECTION_NAME".  You should have a final class field that is initialized with this value.  Scope should probably be similar.